hw/ip/hmac/data/hmac.hjson

```diff
@@ -195,6 +201,24 @@
                 based on currently received message.
                 '''
         }
+        { bits: "2",
+          name: "hash_stop",
```

Comment:
@moidx @jadephilipoom @ballifatih @gdessouky can you take a look at the API changes to double check this is fulfills the requirements of the cryptolib?